app/views/topics/index.html.erb

```diff
@@ -8,7 +8,12 @@
 
 <% content_for :breadcrumbs do %>
   <%= render partial: 'govuk_component/breadcrumbs', locals: @navigation_helpers.breadcrumbs %>
-  <div class="header-context"><!-- Deliberately empty. This will prevent Static from adding a placeholder breadcrumb. --></div>
+  <div class="header-context">
+    <!--
```

Comment:
After https://github.com/alphagov/static/pull/879 is deployed we can get rid of this line entirely (and the others - https://github.com/search?q=org%3Aalphagov+%22Deliberately+empty.+This+will+prevent%22&type=Code)